src/styles/base.scss

```diff
@@ -0,0 +1,59 @@
+body {
+  margin: 0;
+  --column-count: 1;
+}
+
+
+
+.main {
+  padding-top: $main-padding-top;
+  padding-inline: $main-padding-inline;
+  display: grid;
+  grid-template-columns: repeat(var(--column-count), $card-width);
```

Comment:
The variable `$card-width` is used but not defined. Please define this variable to avoid compilation errors.